test/e2e/network/network_policy.go

```diff
@@ -1370,6 +1370,130 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 			})
 		})
 
+		ginkgo.It("should enforce multiple stacked egress network policies with overlapping CIDRs in CIDR and Except [Feature:NetworkPolicy]", func() {
```

Comment:
Thoughts on this:
```
should ensure an IP overlapping both IPBlock.CIDR and IPBlock.Except is allowed [Feature:NetworkPolicy]
```
?

Reply:
how about
``should ensure an IP overlapping both IPBlock.CIDR and IPBlock.Except in multiple, stacked network policies is allowed [Feature:NetworkPolicy]``

or too long?

Reply:
on second thoughts.. your suggestion sounds better.. updated